Formula/python3.rb

```diff
@@ -310,7 +310,22 @@ def post_install
   end
 
   def xy
-    version.to_s.slice(/(3\.\d)/) || "3.6"
```

Comment:
@ilovezfs We need this pre-build.
@yan12125 Yes. This could use `(devel || stable).version.to_s` instead.